### PR TITLE
Update multi-factor-authentication.md

### PR DESCRIPTION
**Pull request by msbemba**
Added info about TAP

Fixes https://github.com/MicrosoftDocs/memdocs/issues/3931

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/enrollment/multi-factor-authentication.md [hunk 1]
```diff
@@ -100,4 +100,7 @@ After you apply and deploy this policy, users will see a one-time MFA prompt whe
 > - iOS/iPadOS devices enrolled via Apple automated device enrollment  
 > - macOS devices enrolled via Apple automated device enrollment  
 >
-> The second device is required because the primary device can't receive calls or text messages during the provisioning process.  
+> The second device is required because the primary device can't receive calls or text messages during the provisioning process.
+>
+> Or you can configure [Temporary Access Pass](/azure/active-directory/authentication/howto-authentication-temporary-access-pass)
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/multi-factor-authentication.md:105]:
```suggestion
> Alternatively, you can configure a [Temporary Access Pass](/azure/active-directory/authentication/howto-authentication-temporary-access-pass).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@lenewsad Please update. Thanks

**Review by lenewsad — COMMENTED** [review]
Hi @msbemba, thanks for the edit. I can't confirm if this is something that device users can use as an alternative option to this policy, so no change to docs. If this is something that the product team identifies as an alternative in the future, we will be sure to add and expand upon it.

**Pull request closed without merging**